src/components/CryptoTraq.js

```diff
@@ -24,16 +30,26 @@ const CryptoTraq = (props) => {
           <SelectionForm
               formLabel={"Select Cryptocurency:"}
               selectionFunc={props.handleCryptoChange}
-              selectForm={<CryptoSelectForm list={props.cryptoList}/>}
+              selectForm={<CryptoSelectForm list={props.cryptoList} delay={styles.delay1} anim={styles.select__glow}/>}
           />
           <p style={styles.arrow}>&rarr;</p>
           <SelectionForm
             formLabel={"Select international currency:"}
             selectionFunc={props.handleIntlChange}
-						selectForm={<IntlSelectForm list={props.intlList}/>}
+						selectForm={<IntlSelectForm list={props.intlList} delay={styles.delay2} anim={styles.select__glow}/>}
 					/>
-          <Button onClick={props.handleExchangeSearch} buttonContent={"Track currency"}/>
+          <Button onClick={props.handleExchangeSearch} buttonContent={"Track currency"} buttonAnimation={styles.trackAnim}/>
         </div>
+				{props.initializing ?
+					<LoadingSpinner
+						loading={props.initializing}
+						content={"Loading currencies"}
+					/> : null}
+				{props.loading ?
+					<LoadingSpinner
+						loading={props.loading}
+						content={'Fetching your results, please hold on.'}
+					/>: null}
```

Comment:
Here I have added the `LoadingSpinner` for when the app is initializing, and also one for when the app is fetching the conversion data.  Both of these can take some time especially on a slower connection and help to let the user know that their input has been received and action is being taken.